index.md

```diff
@@ -13,8 +13,10 @@ This list contains help guides for the M-RETS, broken down by sidebar categories
 ## Certificates
 
 ### [Certificates Dashboard: An Introduction](https://mrets.github.io/Help/Certificates_Dashboard)
+Introduction to the Certificates Dashboard 
 
 ### [Reserve Certificates Overview](https://mrets.github.io/Help/Reserve_Certificates_Overview)
+What is a reserve ? 
```

Comment:
maybe change so it isn't a question- only because the is not how the other article desc. are formatted